### PR TITLE
Removed call to users array

### PR DESCRIPTION
**Pull request by GeekTrainer**
I removed the users array. It was declared at the class level, meaning that it would be tied to one server. In addition, it's not needed as we're only using Passport to simplify the authentication process.

### CODE REVIEW

**Comment by GeekTrainer** [comment]
Made updates to how the resurrect code works. Rather than parsing items out, I simply serialize the address and pass that around as needed.

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "botbuilder": "^3.2.3",
     "passport": "^0.3.2",
     "passport-azure-ad": "2.0.0",
+    "querystring": "^0.2.0",
```
Comment (bnookala) [inline package.json:26]: kind of confused; are you using the builtin `querystring` module in your code? (https://nodejs.org/api/querystring.html)

Comment (GeekTrainer) [reply package.json:26]: I don't know why I did that. :-) That can be removed.

File: app.js [hunk 2]
```diff
@@ -13,143 +14,111 @@ const crypto = require('crypto');
...
-  var address = { id: addressId,
-     channelId: 'webchat',
-     user: { id: userId, name: userId },
-     conversation: { id: conversationId },
-     bot: { id: 'authbot', name: 'authbot' },
-     serviceUrl: 'https://webchat.botframework.com',
-     useAuth: true };
-
-    var continueMsg = new builder.Message().address(address).text("signin?authcode=" + authcode + "&code=" + code + "&name=" + req.user.displayName + "&email=" + req.user.email);
     bot.receive(continueMsg.toMessage());
```
Comment (ritazh) [inline app.js:68]: In this PR, `continueMsg` is not set.

File: app.js [hunk 3]
```diff
@@ -181,154 +138,118 @@ passport.use(new OIDCStrategy(strategy,
...
-          if(authcode && code){
-            session.beginDialog('validateCode', { code : code, authcode : authcode });
-
-          } else {
-            session.replaceDialog('signinPrompt', { invalid: true });
-          }
-      } else {
-          session.replaceDialog('signinPrompt', { invalid: true });
-      }
+    session.userData.loginData = JSON.parse(results.response);
```
Comment (ritazh) [inline app.js:216]: The response here is not a json object.

**Comment by GeekTrainer** [comment]
Closing this because it doesn't work. :-)

**Pull request closed without merging**